packages/sdk/tsup.config.mjs

```diff
@@ -10,4 +10,5 @@ export default defineConfig({
     swap: 'src/swap/index.ts',
     funding: 'src/funding/index.ts',
   },
+  target: 'es2022',
```

Comment:
set target in bundler config instead of `tsconfig`. this is how we do it for the `cli` package and in the `chainflip-product-toolkit`